src/main/java/com/esri/samples/map/read_geopackage/ReadGeoPackageSample.java

```diff
@@ -0,0 +1,190 @@
+        @Override
+        protected void updateItem(Layer layer, boolean bln) {
+
+          super.updateItem(layer, bln);
+          if (layer != null) {
+            if (layer instanceof FeatureLayer) {
+              FeatureLayer featureLayer = (FeatureLayer) layer;
+              GeoPackageFeatureTable featureTable = (GeoPackageFeatureTable) featureLayer.getFeatureTable();
+              setText(featureTable.getDescription());
+            } else if (layer instanceof RasterLayer) {
+              RasterLayer rasterLayer = (RasterLayer) layer;
+              GeoPackageRaster raster = (GeoPackageRaster) rasterLayer.getRaster();
+              setText(raster.getDescription());
+            }
+          } else {
+            setText(null);
+          }
+        }
+
+      };
+
+      mapLayers.setCellFactory(cellFactory);
+      geoPackageLayers.setCellFactory(cellFactory);
+
+      // when you click on a layer in the geopackage layers list view, add it to the map
+      geoPackageLayers.setOnMouseClicked(e -> {
+        if (e.isStillSincePress() && e.getButton() == MouseButton.PRIMARY) {
+          // get selected layer
+          Layer layer = geoPackageLayers.getSelectionModel().getSelectedItem();
+          if (layer != null) {
+            // add it to the map and the top of the map layers list
+            map.getOperationalLayers().add(layer);
+            mapLayers.getItems().add(0, layer);
+            // remove it from the geoPackage layers list
+            geoPackageLayers.getItems().remove(layer);
+          }
+        }
+      });
+
+      // when you click on a layer in the map layers list view, remove it from the map
+      mapLayers.setOnMouseClicked(e -> {
+        if (e.isStillSincePress() && e.getButton() == MouseButton.PRIMARY) {
+          // get selected layer
+          Layer layer = mapLayers.getSelectionModel().getSelectedItem();
+          if (layer != null) {
+            // remove it from the map and the map layers list
+            map.getOperationalLayers().remove(layer);
+            mapLayers.getItems().remove(layer);
+            // add it back to the geoPackage layers list
+            geoPackageLayers.getItems().add(layer);
+          }
+        }
+      });
+
+      // read the raster and feature layers from the geoPackage and show them in the list view
+      File geoPackageFile = new File("./samples-data/auroraCO/AuroraCO.gpkg");
+      GeoPackage geoPackage = new GeoPackage(geoPackageFile.getAbsolutePath());
+      geoPackage.loadAsync();
+      geoPackage.addDoneLoadingListener(() -> {
+        geoPackage.getGeoPackageRasters().forEach(r -> {
+          RasterLayer rasterLayer = new RasterLayer(r);
+          // make the raster layer semi-transparent so we can see layers below it
+          rasterLayer.setOpacity(0.5f);
+          geoPackageLayers.getItems().add(rasterLayer);
+        });
+        geoPackage.getGeoPackageFeatureTables().forEach(t -> geoPackageLayers.getItems().add(new FeatureLayer(t)));
```

Comment:
This is the part that I was talking about putting before `getGeoPackageRasters` to align with the README.